GeosCore/hco_utilities_gc_mod.F90

```diff
@@ -2573,6 +2573,12 @@ SUBROUTINE Get_Boundary_Conditions( Input_Opt, State_Chm, State_Grid, &
       ! Free pointer
       SpcInfo => NULL()
 
+      ! Each time the boundary conditions are read, they have no longer been perturbed
+      ! This value is sometimes set to True in set_boundary_conditions_mod.F90
+      IF (Input_Opt%ITS_A_CH4_SIM .AND. Input_Opt%PerturbCH4BoundaryConditions ) THEN
```

Comment:
I recommend adding `.OR. Input_OPt%ITS_A_CARBON_SIM` here (enclosed in parentheses with ITS_A_CH4_SIM)